app/models/server/models/_Base.d.ts

```diff
@@ -0,0 +1,16 @@
+export declare class Base {
+	model: any;
+
+	constructor(nameOrModel: string);
+
+	tryEnsureIndex(...args: any[]): any;
+
+	findOne(...args: any[]): any;
+	static findOne(...args: any[]): any;
+	
+	insert(...args: any[]): any;
+	static insert(...args: any[]): any;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static method not allowed after declaration of public instance method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/app/RocketChat/Rocket-Chat/pullRequest?prid=4420492)